src/Config.cpp

```diff
@@ -38,11 +38,13 @@ with this program. If not, see <https://www.gnu.org/licenses/>
 #define PARAM_FIRSTLOAD "first_load"
 #define PARAM_ENABLED "server_enabled"
 #define PARAM_PORT "server_port"
+#define PARAM_HOST "server_host"
 #define PARAM_ALERTS "alerts_enabled"
 #define PARAM_AUTHREQUIRED "auth_required"
 #define PARAM_PASSWORD "server_password"
 
 #define CMDLINE_WEBSOCKET_PORT "websocket_port"
+#define CMDLINE_WEBSOCKET_HOST "websocket_host"
```

Comment:
I think that I would like this to use `address` instead of `host`. `address` would likely be less ambiguous.

Reply:
I used host because that's what websocketpp called it, but agreed, address is a better name.